_developer_documentation/_BP_design.info

```diff
@@ -37,9 +37,6 @@ Gameplay:
 -----------------------------------------------------------------
 IN PROGRESS:
 -----------------------------------------------------------------
-IMMEDIATE:
-* Rename Proto-Carpathian to Balko-Danubian heritage (Proto-Carpathian being a Con-Lang)
```

Comment:
What? Proto-Carpathian was reconstructed specifically because there were no attested remains. That doesn't mean the name is restricted to that reconstruction. I don't understand the justification. Paleo-Balkan doesn't cover the same scope as Proto-Carpathian, unless that is what you wanted to represent?

Reply:
Well, the Carpathians at best really only cover the Dacians and Thracians; they are a far distance from the Dinaric Alps or the Balkan Mountains for the Thracians, Illyrians, Macedonians, and Albanians. OTOH, the general "Balkan" geographic roughly covers the geographic extent of all the Cultures in that heritage group.

I don't think Proto-Carpathian provides the same geographic coverage.